### PR TITLE
added travis config file

### PR DESCRIPTION
**Pull request by jflewis**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+- "5"
+- "node"
```
Comment (scottgonzalez) [inline .travis.yml]: Remove this.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+notifications:
+  email:
+  - sgonzalez@clippermagazine.com
+  - kpollich@clippermagazine.com
```
Comment (scottgonzalez) [inline .travis.yml]: Remove email notifications.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+- "5"
+- "4"
+notifications:
+  slack:
+    secure: M900aBc4VfFvIhHVghn7rJM488hWuaLKCayldjJEB03qYGvGJBL5gdarCKmijJpw5SLkk7JvthhMOpaK/gJgWLrybS+HE8AFqsGySS79vXklYoRJ1nVcGETWsHVu3DG3qz2IRmv9ZtE48hcfIDhCkuZsn6ddV7frHxv1ObzuXGLIXB+OJW2gjfnyJWxxdi93WwuDsCkHbKOWebAvuHfLgaCLSgS1+ncRnx/G85yvtwCIFqZeaSDXjWZ2dProcSBP2OWmPHoive2F6pWjbePdDsb6C82SKlKqYSwH9rJuowbm0bFya3qFt0LKBa30kkgc/cjXIESoy8423v3ssecW28/Und8e3jWioY87z7KLuNEnfYKo2RuqwWTj+rLTzvVOc8RCmP1sNNy/eMzeE8wXYv/b3HRxDEvfiQJd2AS7F4rE7wYEkkdWmu76ElVmgYI1ZYfUmJCajl8s/UMkAn5mFRXU6lQwtitoX3L9POcZFIJ2Cw6beI8KbtpYxVIqt1/MxIibi9s4MKru9dgSXYad3xwwy9r9G88bZo+n2aPo2lmwCzgNMLp9NbJDNpDh08iA0lvPnC0lHUYL7agV5eSonUWqCeFSaT/AgrsOb64xjgmF/0Wd3lVZL9xmcAmCfvv21EhDDjOXLsi9DkQFWjimqt3aURotwEjRe+023yBV/Zg=
```
Comment (kpollich) [inline .travis.yml]: Is this something that we should worry about being public?

Comment (scottgonzalez) [inline .travis.yml]: Well, it doesn't leak anything. People might be upset that an open source project is using a private Slack channel though.

Comment (kpollich) [inline .travis.yml]: True that. Maybe we should remove Slack for now. The notifications aren't important, imo.

Comment (jflewis) [inline .travis.yml]: @kpollich I encrypted the data using the travis CLI. But i agree with not wanting to upset anyone so I took it out

**Pull request closed without merging**